docs/store/Makefile

```diff
@@ -1,20 +1,37 @@
-images = 1.svg.png 2.svg.png 3.svg.png 4.svg.png
+# Inspired from https://stackoverflow.com/a/32535737/2367848
+locales     := fr-FR en-US
```

Comment:
Can we list the locales from the fastlane folder instead?

Reply:
Okay, but this means we need to add the .json translation file for screenshots as new locales are added. Not so hard but we might see weird make errors if when it will be missing for a new locale. I think we can wait until this happens and advise accordingly then.